### PR TITLE
Fix install instructions.

### PR DESCRIPTION
**Pull request by progval**
Closes GH-745.

### CODE REVIEW

File: docs/Running-Mastodon/Production-guide.md [hunk 1]
```diff
@@ -200,7 +199,7 @@ After=network.target
 Type=simple
 User=mastodon
 WorkingDirectory=/home/mastodon/live
-Environment="NODE_ENV=production"
+Environment="RAILS_ENV=production"
```
Comment (Gargron) [inline docs/Running-Mastodon/Production-guide.md]: NODE_ENV != RAILS_ENV, this is a Node application, so you don't need to set RAILS_ENV here

**Comment by Gargron** [comment]
Just to confirm before merging - the removal of `CREATE DATABASE mastodon_production OWNER mastodon;` is intentional, yes? I forget, does `rails db:setup` create the database or why?

File: docs/Running-Mastodon/Production-guide.md (hunk 1 above)
Comment (ProgVal) [inline docs/Running-Mastodon/Production-guide.md]: Yeah, I messed up this one. Commit removed from the PR.

**Comment by ProgVal** [comment]
> Just to confirm before merging - the removal of `CREATE DATABASE mastodon_production OWNER mastodon;` is intentional, yes?

Yes.

> I forget, does `rails db:setup` create the database or why?

Yes, that's the problem I submitted in #745. I don't know if it's a bug or not, it's my first time debugging a Ruby app.

**Pull request merged**